css/media.css

```diff
@@ -0,0 +1,400 @@
+@media (max-width: 1220px) {
+  .welcome-page {
+    width: 100%;
+    margin: 0 auto;
+  }
+}
+
+@media (max-width: 1036px) {
+  .header__menu_link {
+    font-size: 12px;
+    line-height: 15px;
+  }
+
+  .link__item {
+    font-size: 12px;
+    line-height: 15px;
+  }
+}
+
+@media (max-width: 960px) {
+  .car {
+    background-position-x: -40px;
+  }
+}
+
+@media (max-width: 856px) {
+  .header__menu_link {
+    display: none;
+  }
+
+.welcome-page {
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5566861)